webtodotxt.py

```diff
@@ -18,7 +18,8 @@
 app.config['LANGUAGES'] = {
     'en': 'English',
     'fr': 'Français',
-    'pt': 'Portuguese'
+    'pt': 'Portuguese',
+    'de': 'German'
```

Comment:
should it be "German" or in native language "Deutsch" here?

Reply:
Native language please.